website/docs/getting-started/schema-field.md

```diff
@@ -172,8 +172,10 @@ If your schema has a different or complicated way of loading, you can point to a
 
 ```yml
 schema:
-  - ./my-schema-loader.js:
-      noPluck: true
+  - http://localhost:3000/graphql:
```

Comment:
Reverted the change to the docs here. While CodeFileLoader can be used here with the `noPluck` option, it doesn't actually work with functions (see [here](https://github.com/ardatan/graphql-tools/blob/master/packages/loaders/code-file/src/exports.ts#L60-L88))